pkg/openthread/contrib/platform_uart.c

```diff
@@ -13,34 +13,164 @@
  * @brief       Implementation of OpenThread UART platform abstraction
  *
  * @author      Jose Ignacio Alamos <jialamos@uc.cl>
+ * @author      Baptiste Clenet <bapclenet@gmail.com>
  * @}
  */
 
 #include <stdint.h>
 #include <stdio.h>
 
 #include "periph/uart.h"
+#include "openthread/types.h"
 #include "openthread/platform/uart.h"
+#include "ot.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#define UART_OPENTHREAD_BAUDRATE            (115200U)
```

Comment:
This won't work on iotlab-m3 since it works at 500000. You should add this define in the openthread configuration file. Same for `UART_OPENTHREAD`.